### PR TITLE
Email validation and Password Reset

### PR DESCRIPTION
**Pull request by Singletoned**

### CODE REVIEW

**Comment by Singletoned** [comment]
I managed to push the fix for that before you pasted your comment

**Comment by nmenezes0** [comment]
> I managed to push the fix for that before you pasted your comment

Thanks!

File: .envs/api [hunk 1]
```diff
@@ -1,10 +1,12 @@
 DEBUG=True
 DJANGO_SETTINGS_MODULE=ellandi.settings
 DJANGO_SECRET_KEY="django-insecure-&3_9ebzl!$@-vv=v=u@6h=lm@abd69u&cfs4lj0ml0v$thw*f3"
+VCAP_APPLICATION='{"application_uris": ["localhost:8001"]}'
```
Comment (nmenezes0) [inline .envs/api:4]: Should this be `8000`?
(I think so, otherwise the links for resetting password etc. don't work.)

**Comment by nmenezes0** [comment]
Not particularly important - should the schema show the parameters for the `password-reset`? 
image

**Review by nmenezes0 — APPROVED** [review]
Have left 3 minor comments/questions.

File: .envs/api (hunk 1 above)
Comment (Singletoned) [reply .envs/api:4]: Well, we'll see how @rottitime wants to handle it, but theoretically the links should go to the web frontend, not to the API

Comment (nmenezes0) [reply .envs/api:4]: In which case - it's fine then. (I maybe didn't know what it was supposed to be doing.)

**Comment by Singletoned** [comment]
Yes, we will return any errors in that format, but we can't return an error saying that the user doesn't exist as it leaks information.

File: .envs/api (hunk 1 above)
Comment (rottitime) [reply .envs/api:4]: @nmenezes0  @Singletoned I would be happy with the links going to FE

**Review by rottitime — APPROVED** [review]

**Pull request merged**